public/app/core/utils/accessControl.ts

```diff
@@ -1,9 +1,9 @@
 import config from '../../core/config';
 
 // addAccessControlQueryParam appends ?accesscontrol=true to a url when accesscontrol is enabled
-export function addAccessControlQueryParam(url: string): string {
+export function addAccessControlQueryParam(params: Record<string, any>): Record<string, any> {
```

Comment:
```suggestion
export function addAccessControlQueryParam(params={}) {
```
I think this function can be simplified, which would also avoid calling it with an empty object. 

Also not sure about the naming, since after this change it doesn't add params but returns them, so maybe `getAccessControlQueryParams` or just `accessControlQueryParams` would be better? The comment should probably be updated as well.